packages/server/src/routes/auth/login.ts

```
@@ -1,11 +1,13 @@
-import { createRoute, RouteHandler, z } from "@hono/zod-openapi";
-import { ENV_TYPES, loginFields } from "@repo/zod-types";
+import { createRoute, z } from "@hono/zod-openapi";
+import { loginFields } from "@repo/zod-types";
 import { env } from "hono/adapter";
-import { AuthTable } from "./handlers";
+import { AuthTable, ClientTable } from "./handlers";
 import { checkPassword } from "../../utils/bcrypt";
 import { generateToken } from "../../jwt_token";
 import { setCookie } from "hono/cookie";
 import { COOKIES } from "../../env/cookies";
+import { AppRouteHandler } from "../../base/type";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)